Sources/TensorFlow/Bindings/generate_wrappers.py

```diff
@@ -76,19 +76,21 @@
   ({types_pb2.DT_UINT8,
     types_pb2.DT_UINT16,
     types_pb2.DT_UINT32,
-    types_pb2.DT_UINT64}, 'UnsignedInteger & TensorFlowScalar'),
+    types_pb2.DT_UINT64}, 'TensorFlowUnsignedInteger'),
```

Comment:
It's a little weird to have an `Unsigned` protocol because Swift standard library doesn't. Are there any TensorFlow ops that only take unsigned integers?

Reply:
Good point. There aren't actually. I had just left that there because it existed before I started modifying the op generation script, but given this, I'll just remove it.

Reply:
I guess there is `encodePng` and `decodePng`, but we can remove it since there's only two of them.

Reply:
Done.